back-end/www/models/model.py

```diff
@@ -42,3 +42,56 @@ class User(db.Model):
     def __repr__(self):
         return "<User id=%r created_at=%r client_id=%r client_type=%r>" % (
                 self.id, self.created_at, self.client_id, self.client_type)
+
+
+class Location(db.Model):
+    """
+    Class representing a location.
+
+    Attributes
+    ----------
+    id : int
+        Unique identifier as primary key.
+
```

Comment:
no need to put blank lines in the middle (for the format please check https://numpydoc.readthedocs.io/en/latest/format.html)